GamebookEngine/Views/Game List/GameListView.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  GameListView.swift
+//  GamebookEngine
+//
+//  Created by Brad Root on 4/7/23.
+//  Copyright © 2023 Brad Root. All rights reserved.
+//
+
+import SwiftUI
+
+struct GameCard: View {
+    let game: Game
+
+    var body: some View {
+        HStack {
+            VStack {
+                Text(game.name)
+                    .multilineTextAlignment(.leading)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .font(.title)
+                    .fixedSize(horizontal: false, vertical: true)
+                    .padding(EdgeInsets(top: 22, leading: 22, bottom: 4, trailing: 22))
+
+                Text(game.author)
+                    .font(.headline)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .foregroundColor(.secondary)
+                    .padding(EdgeInsets(top: 0, leading: 22, bottom: 8, trailing: 22))
+
+                Text(game.about ?? "")
+                    .font(.subheadline)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .padding(EdgeInsets(top: 0, leading: 22, bottom: 8, trailing: 22))
```

Comment:
This can help with text getting truncated: 

https://stackoverflow.com/questions/56507497/views-compressed-by-other-views-in-swiftui-vstack-and-list/57677746#57677746

```
Text(content)
                .fixedSize(horizontal: false, vertical: true)
                .frame(maxWidth: .infinity, alignment: .leading)
                .font(.subheadline)
                .padding(EdgeInsets(top: 0, leading: 22, bottom: 8, trailing: 22))
```